temporalio/lib/temporalio/worker.rb

```diff
@@ -299,6 +300,7 @@ def self.default_illegal_workflow_calls
     # @param build_id [String] Unique identifier for the current runtime. This is best set as a unique value
     #   representing all code and should change only when code does. This can be something like a git commit hash. If
     #   unset, default is hash of known Ruby code.
+    #   Exclusive with `deployment_options`.
```

Comment:
Mentioned internally, but I'd be ok just removing `build_id` and `use_worker_versioning` since setting them directly is deprecated as I understand it (and we're not GA)

Reply:
Cool, forgot we had discussed that lol